### DISCUSSION TITLE
[vcpkg] Integration with Makefile-like

### DISCUSSION THREAD

**Discussion by zoeleu** [microsoft/vcpkg#17874] [Ideas]
****

**Is your feature request related to a problem? Please describe.**
No.

**Proposed solution**
Example: vcpkg show includes sdl2 cpr
/home/8x13b/vcpkg/ports/sdl2/include
/home/8x13b/vcpkg/ports/cpr/include

**Describe alternatives you've considered**
using cmake, but integrating vcpkg is better

**Comment by JackBoosY** [comment]
Hi @matteoturini, I think that's not needed. because vcpkg support integration to your working environment. See [documentation](https://github.com/microsoft/vcpkg/blob/master/docs/users/integration.md).
And the include path should be _VCPKG_ROOT/installed/TRIPLET/include_.

**Comment by zoeleu** [comment]
Oh, thanks! I didn't find that in the docs.

**Comment by zoeleu** [comment]
I still think it would be nice to make a Makefile-like integration.
